README.md

```diff
@@ -31,12 +31,6 @@ You can run your application in dev mode that enables live coding using:
 
 You can run tests from your IDE or via Maven. Simply run `./mvnw test ` or `./mvnw package`
 
-## Tkit quarkus
-
-Adding tkit libs to our project gives us several new features. Check the logs for example, all our business methods are now logged and timed. 
-The REST API now handles exceptions gracefully(as JSON response), and we get server side pagination with very little effort. 
-Our tests are now real integration tests with real postgres DB, and are stuitable for CI envs.
-
```

Comment:
do we have some documentation in our docs adding this special kind of overview of tkit libs and their key features given a code example to make people aware of the libs and let them take the choice to choose or not to choose?
If not, we might want to create a ticket with clear description on what we would expect and pass it to Matus to get the docs possibly directly from the maintainers.

Reply:
No, but there is already a ticket to add some documentation about tkit in devon4j https://github.com/devonfw/devon4j/issues/486